pkg/cmd/roachtest/tests/cdc_bench.go

```diff
@@ -56,15 +56,21 @@ const (
 	// practice it can.
 	cdcBenchColdCatchupScan cdcBenchScanType = "catchup-cold"
 
-	cdcBenchNoServer        cdcBenchServer = ""
-	cdcBenchProcessorServer cdcBenchServer = "processor" // legacy processor
+	cdcBenchNoServer cdcBenchServer = ""
+	// The legacy processor was removed in 25.1+. This is left in
+	// place for the roachtests for now to avoid chaning the same
```

Comment:
What does keeping this around achieve? As of this commit we're no longer running the legacy flavor roachtest (and even if we did it would be run with the scheduler). `roachperf` doesn't care whether we define an unused constant here or not. I think we should remove this unless there's something here that I'm not seeing.